src/Client.php

```diff
@@ -141,21 +141,59 @@ public function create(AutotaskObjects\Entity $obj)
         $params = new AutotaskObjects\CreateParam($obj);
         return $this->_call('create', array($params));
     }
+    
+    public function bulkCreate(array $objs)
+    {
+        if (count($objs) > 200) {
+            throw new \Exception('You can only execute a bulk create on a max of 200 objects per request');
+        }
+        $createObjs = null;
+        foreach ($objs as $obj) {
+            if (!$createObjs){
+               $createObjs = new AutotaskObjects\CreateParam($obj);
+            } 
+            else {
```

Comment:
Sorry -- I am a stickler for code style.

`else` should be on the same line as the ending `}` for the if, with a space separating it.
```
if (something) {
} else {
}
```

Same goes for the if/else block in bulkUpdate